keps/sig-node/2837-pod-level-resource-spec/README.md

```diff
@@ -1172,6 +1210,183 @@ back to aggregating container requests.
   size of the pod's cgroup. This means the pod cgroup's resource limits will be
   set to accommodate both pod-level requests and pod overhead.
 
+#### Hugepages
+
+With the proposed changes, support for hugepages(with prefix hugepages-*) will be extended to the pod-level resources specifications, alongside CPU and memory. The hugetlb cgroup for the
+pod will then directly reflect the pod-level hugepage limits, if specified, rather than using an aggregated value from container limits. When scheduling, the scheduler will 
+consider hugepage requests at the pod level to find nodes with enough available
+resources.
+
+Containers will still need to mount an emptyDir volume to access the huge page filesystem (typically /dev/hugepages).  This is the standard way for containers to interact with huge pages, and this will not change. 
```

Comment:
This sounds confusing; `/dev/hugepages` isn't normally present within an empty directory (per `emptyDir`). Maybe clarify what we mean here.